components/Workspaces/SearchReposTable.tsx

```diff
@@ -0,0 +1,98 @@
+import { BiBarChartAlt2 } from "react-icons/bi";
+import { FaPlus } from "react-icons/fa6";
+import { useState } from "react";
+import Button from "components/atoms/Button/button";
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "components/shared/Table";
+import { Avatar } from "components/atoms/Avatar/avatar-hover-card";
+import Checkbox from "components/atoms/Checkbox/checkbox";
+import Search from "components/atoms/Search/search";
+
+interface SearchedReposTableProps {
+  repositories: {
+    owner: string;
+    name: string;
+  }[];
+  onFilter: () => void;
+  onSelect: () => void;
+}
+
+const EmptyState = ({ onAddRepos }: { onAddRepos: () => void }) => {
+  return (
+    <div className="grid place-content-center gap-4 my-8">
+      <BiBarChartAlt2 className="border rounded p-1 w-12 h-12 mx-auto" />
+      <div className="grid w-max max-w-sm mx-auto">
+        <span className="text-center text-lg font-semibold mb-2">Add repositories to track</span>
+        <p className="text-sm text-center">
+          Search and select the repositories you want to track and get insights on your entire Github ecosystem
+        </p>
+      </div>
+      <Button variant="primary" className="w-max mx-auto" onClick={onAddRepos}>
+        <FaPlus className="mr-2 text-lg" />
+        Add repositories
+      </Button>
+    </div>
+  );
+};
+
+export const SearchedReposTable = ({
+  repositories = [],
+  onFilter,
+  onSelect: onSelectRepo,
+}: SearchedReposTableProps) => {
+  const [filteredRepositories, setFilteredRepositories] = useState(repositories);
+
+  return (
+    <div className="border border-light-slate-7 rounded-lg">
+      <Table className="not-sr-only">
```

Comment:
It might look odd that there is a table with a header and now rows and then the same table header with rows.

The reason being is to make the header fixed and the rows scrollable, I had to do this hack as you can't make a <tbody /> element scrollable. I've wrapped the second table in a div with a fixed height and overflow scroll Y. Visually, only one header will be seen. For a screen reader, they will only see the one table with header and rows.